develop-docs/self-hosted/email.mdx

```diff
@@ -11,14 +11,18 @@ description: Set up and configure email notifications for your self-hosted Sentr
 
 ## Outbound Email
 
-Self-hosted Sentry ships with a built-in outgoing SMTP server powered by [exim4](https://hub.docker.com/r/tianon/exim4). The default configuration is set to use this server. All you need to do is to set a valid address for `mail.from` setting in `config.yml` and the [FQDN](https://en.wikipedia.org/wiki/Fully_qualified_domain_name) of your Sentry instance for `SENTRY_MAIL_HOST` in `.env`. Keep in mind that if you start sending too many emails to public addresses, your new server may get marked as a spammer and banned.
+It is recommended to use an external SMTP server due to various sender requirements implemented by major email providers, but if you are testing or running a self-hosted Sentry instance on a local network, you can use the built-in SMTP serverpowered by [exim4](https://hub.docker.com/r/tianon/exim4). The default configuration is set to use this server. All you need to do is to set a valid address for `mail.from` setting in `config.yml` and the [FQDN](https://en.wikipedia.org/wiki/Fully_qualified_domain_name) of your Sentry instance for `SENTRY_MAIL_HOST` in `.env`.
 
-If you want to use an external SMTP server you can set the relevant `mail.*` settings in `config.yml` file and ignore the built-in SMTP server. Refer to our [email service documentation](/backend/email/) for all the details on what each setting means and does. If your MX records usually resolve to internal private network mail handlers, see [this ticket](https://github.com/getsentry/self-hosted/issues/1202) for some pointers.
+If you want to use an external SMTP server you can set the relevant `mail.*` settings in `config.yml` file and ignore the built-in SMTP server. If you don't have an external SMTP server, yet you need to see sent emails, you can set `mail.backend` to `console` on your `config.yml` file. Refer to our [email service documentation](/backend/email/) for all the details on what each setting means and does.
+
+If your MX records usually resolve to internal private network mail handlers, see [this ticket](https://github.com/getsentry/self-hosted/issues/1202) for some pointers.
 
 <Alert title="Warning" level="warning">
   Because of the way configuration is layered, if you update <code>mail</code> settings through the web interface, you will need to also comment out the <code>mail.host: 'smtp'</code> default in your <code>config.yml</code> in order for your desired settings to be picked up.
 </Alert>
 
+Sentry does not support sending outbound email with any other protocol than SMTP (examples: Sendgrid's API, AWS SES API, Mailgun API). Usually, those services provide SMTP relay that you can use.
```

Comment:
~~I thought we had support for Mailgun.~~ Okay, looks like that's only for inbound.